Classes/Domain/Model/Cart/Cart.php

```diff
@@ -1280,7 +1280,8 @@ private function calcTax()
      */
     public function reCalc()
     {
-        $this->calcGross();
+	    $this->calcCount();
```

Comment:
Do you habe an example for the wrong number of products in cart? I can imagine, when that happens, but would like to check the behavior.